### PR TITLE
STY: Completed funding institution list

### PR DESCRIPTION
**Pull request by rstoneback**
Addresses #1168

# Description

Completes the list of funding institutions from most awesome down to NASA

## Type of change

Please delete options that are not relevant.

- This change requires a documentation update

# How Has This Been Tested?

Visual inspection of the RTD generated docs

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the release
checklist on the wiki: https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

**Comment by jklenzing** [comment]
TODO: update with single acknowledgments file and point to this from the individual files

**Comment by rstoneback** [comment]
I added the acknowledgements and added a test sentence for pointing to acknowledgements.txt

File: acknowledgements.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+NASA Scintillation Prediction Observations Research Task (SPORT)
+
+Programs
+--------
+NSF 125908, AGS-1651393
+NASA NNX10AT02G, NNH20ZDA001N-LWS, 80NSSC18K120, and 80NSSC21M0180
+NASA Space Precipitation Impacts (SPI) project
+Naval Research Laboratory N00173191G016 and N0017322P0744
+
+
```
Comment (rstoneback) [inline acknowledgements.txt:27]: I'm unsure when to use ONR vs NRL. I believe my contracts were NRL but normally I see folks using ONR.

**Comment by rstoneback** [comment]
Apologies for stringing out the pushes. I am out of practice :)

File: docs/index.rst [hunk 2]
```diff
@@ -28,4 +28,6 @@ Welcome to pysat's documentation!
 .. admonition:: DISTRIBUTION STATEMENT A: Approved for public release.
 		Distribution is unlimited.
 
-    This work was supported by the Office of Naval Research.
+Funding
+=======
+.. include:: ../acknowledgements.txt
```
Comment (aburrell) [inline docs/index.rst:33]: Recommend including funding in the main doc, perhaps within "citing.rst" or as it's own section.

File: pysat/utils/_core.py [hunk 3]
```diff
@@ -5,7 +5,8 @@
 #
 # DISTRIBUTION STATEMENT A: Approved for public release. Distribution is
 # unlimited.
-# This work was supported by the Office of Naval Research.
+# This work was supported by the institutions, missions, and programs listed in
+# acknowledgements.txt.
```
Comment (aburrell) [inline pysat/utils/_core.py:9]:
```suggestion
```

**Review by aburrell — CHANGES_REQUESTED** [review]
We could consider finding a way to pipe the acknowledgements.txt lines into zenodo using the "notes" section.

File: acknowledgements.txt [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+ - NOAA Constellation Observing System for Meteorology Ionosphere and Climate (COSMIC-2)
+ - NASA Ionospheric Connections Explorer (ICON)
+ - NASA Scintillation Observations and Response of the Ionosphere to Electrodynamics (SORTIE)
+ - NASA Scintillation Prediction Observations Research Task (SPORT)
+
+Programs
+--------
+ - NSF 125908, AGS-1651393
+ - NASA NNX10AT02G, NNH20ZDA001N-LWS, 80NSSC18K120, and 80NSSC21M0180
+ - NASA Space Precipitation Impacts (SPI) project
```
Comment (jklenzing) [inline acknowledgements.txt:24]:
```suggestion
 - NASA Space Precipitation Impacts (SPI) project at Goddard Space Flight Center through the Heliophysics Internal Science Funding Model.
```

File: acknowledgements.txt [hunk 5]
```diff
@@ -0,0 +1,25 @@
+The following institutions, missions, and programs have provided funding
+for pysat development.
+
+Institutions
+------------
+ - Cosmic Studio
```
Comment (rstoneback) [inline acknowledgements.txt:6]:
```suggestion
  - The Catholic University of America (CUA)
  - Cosmic Studio
```

File: acknowledgements.txt [hunk 6]
```diff
@@ -0,0 +1,25 @@
+The following institutions, missions, and programs have provided funding
```
Comment (rstoneback) [inline acknowledgements.txt:1]:
```suggestion
Funding
======
The following institutions, missions, and programs have provided funding
```

File: acknowledgements.txt [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+ - NASA Ionospheric Connections Explorer (ICON)
+ - NASA Scintillation Observations and Response of the Ionosphere to Electrodynamics (SORTIE)
+ - NASA Scintillation Prediction Observations Research Task (SPORT)
+
+Programs
+--------
+ - NSF 125908, AGS-1651393
+ - NASA NNX10AT02G, NNH20ZDA001N-LWS, 80NSSC18K120, and 80NSSC21M0180
+ - NASA Space Precipitation Impacts (SPI) project at Goddard Space Flight Center through the Heliophysics Internal Science Funding Model.
+ - Naval Research Laboratory N00173191G016 and N0017322P0744
```
Comment (rstoneback) [inline acknowledgements.txt:25]:
```suggestion
 - Naval Research Laboratory N00173191G016 and N0017322P0744
 
Disclaimers
=========
Any opinions or actions taken by the listed funding institutions are those of the institutions and do not necessarily reflect the views of the pysat development team or individual authors. Any opinions, findings, and conclusions or recommendations expressed in this material are those of the author(s) and do not necessarily reflect the views of the funding agencies.
```

File: acknowledgements.txt [hunk 8]
```diff
@@ -0,0 +1,25 @@
+The following institutions, missions, and programs have provided funding
+for pysat development.
+
+Institutions
+------------
```
Comment (rstoneback) [inline acknowledgements.txt:5]: The loss of awesomeical ordering will be felt for some time 👎

**Review by aburrell — COMMENTED** [review]
I would recommend moving 'acknowledgements.txt' to 'ACKNOWLEDGEMENTS.md', to be consistent with the naming conventions of the rest of the informative files in this directory.

**Comment by rstoneback** [comment]
> We could consider finding a way to pipe the acknowledgements.txt lines into zenodo using the "notes" section.

I've found the notes section of the .zenodo.json but .json itself doesn't have functionality to load a file. Zenodo, according to its docs,  interprets the notes portion of the .json as a string (html supported). Within html we'd need something like javascript to actually load the file, presuming Zenodo would allow that code to access files....   In other words, I currently don't see how to connect the acknowledgments into zenodo unless we manually copy with each release.

**Review by aburrell — APPROVED** [review]

**Pull request merged**